HyperwalletUISDK.xcodeproj/project.pbxproj

```diff
@@ -1051,7 +1052,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 86FDS64Z83;
```

Comment:
I have removed development team from the master. Let's keep it that way and not have development team in the commits

Reply:
Sure